app/views/candidates/_candidate_details.html.erb

```diff
@@ -0,0 +1,47 @@
+<section class="h-100">
+  <div class="container h-100">
+    <div class="row justify-content-sm-center h-100">
+      <div class="col-xxl-4 col-xl-5 col-lg-5 col-md-7 col-sm-9">
+        <div class="text-center my-5">
+          <h2> <%= @candidate.persisted? ? "Edit Candidate Details" : "Add candidate details" %></h2>
+        </div>
+        <div class="card shadow-lg">
+          <div class="card-body p-5">
+            <h3 class="fs-4 card-title fw-bold mb-4">Candidate Details</h3>
+            <%= form_with model: @candidate, url: url, local: true do |f| %>
```

Comment:
question: do we need to pass url here, I think it should work without url as well.